lib/ex_unit/test/ex_unit/assertions_test.exs

```diff
@@ -74,6 +76,15 @@ defmodule ExUnit.AssertionsTest do
     end
   end
 
+  test "assert exposes sigil variables in matches" do
```

Comment:
Instead of "sigil variables", what if we call it "nested macro variables", since this is a general issue and not a sigil one?